### ISSUE TITLE
Line breaks in complex selectors using `&` warn of invalid property

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#46]
I use the `&` (parent reference) in Stylus nesting to simplify my CSS naming convention. When creating complex selectors (like styling hyperlinks for :link, :visited, :active, etc.) and using the parent reference, AND using line breaks, Stylint warns of an invalid property when `valid: true`.

property.styl
```stylus
.block1 {
    // Passes just fine
    &--link, &--link:link {
        color: #000;
    }
}

.block2 {
    // Throws warning
    &--link,
    &--link:link {
        color: #000;
    }
}
```

property.json
```json
{
    "alphabetical": true,
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": true,
    "efficient": true,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "parenSpace": false,
    "placeholders": true,
    "semicolons": false,
    "universal": true,
    "valid": true,
    "whitespace": true,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Warning:
```bash
Warning:  property is not valid
File: property.styl
Line: 10: &--link,
```

**Comment by rossPatton** [comment]
im gonna push up a potential fix for this soon

**Comment by rossPatton** [comment]
this is up now, if you pull the latest

**Comment by awayken** [comment]
It looks like this is fixed in the latest pull. Thanks!
